src/util/logger.js

```diff
@@ -0,0 +1,11 @@
+import videojs from 'video.js';
+
+const logger = (source) => {
+  if (videojs.log.debug) {
+    return videojs.log.debug.bind(videojs, '[VHS]', `${source} >`);
+  }
+
+  return function() {};
```

Comment:
Do we want to fall back to `videojs.log` when `videojs.log.debug` does not exist?

Reply:
That would then print out the logging by default

Reply:
ah ya we don't want that